Inbound/0x0055.py

```diff
@@ -50,11 +51,11 @@ def decode_mail_info():
         for j in range(itemCount):
             with Node("AttachedItem " + str(j)):
                 decode_mail_attach_item()
+    add_long("Mesos")
+    add_long("LastPurchaseTimestamp?")
+    add_long("BlueMerets")
     add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
+    add_long("RedMerets")
     add_long("Unknown")
```

Comment:
I wonder if these are also meret values, there's like 5 different meret types.

Reply:
Just tried it out. Doesn't look like it is.

Reply:
thanks for confirming